### DISCUSSION TITLE
部分刮削源刮削的元数据类型风格缺失

### DISCUSSION THREAD

**Discussion by YuleLee-652987** [metatube-community/jellyfin-plugin-metatube#347] [Q&A]
尤其是ARZON，刮削的元数据没有类型风格。

**Comment by YuleLee-652987** [comment]
ARZON刮削源会缺少类型风格，AVABASE刮削源会缺少剧情概括，选择刮削缺少的数据好像只会依据原来的刮削源刮削，不会更换其它刮削源补充缺失的数据。

**Reply by xjasonlyu** [reply]
对的，目前就是不会去补充的，因为有时候不能判断两个数据源一定是同一个影片
